.github/services/webdav/nginx_with_redirect/action.yml

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: nginx_with_redirect
+description: "Behavior test for nginx_with_redirect in nginx"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup webdav in nginx_with_redirect
+      shell: bash
+      working-directory: fixtures/webdav
+      run: docker compose -f docker-compose-webdav.yml up -d
+    - name: Setup
+      shell: bash
+      run: |
+        cat << EOF >> $GITHUB_ENV
+        OPENDAL_WEBDAV_ENDPOINT=tcp://127.0.0.1:8081
```

Comment:
Endpoint should be `http://127.0.0.1:8081`

Reply:
OK, fixed it.